aeron-client/src/main/c/util/aeron_fileutil.c

```diff
@@ -530,25 +530,33 @@ int aeron_map_existing_log(aeron_mapped_raw_log_t *mapped_raw_log, const char *p
 
 int aeron_map_raw_log_close(aeron_mapped_raw_log_t *mapped_raw_log, const char *filename)
 {
-    int result = 0;
-
-    if (mapped_raw_log->mapped_file.addr != NULL)
+    if (!aeron_map_raw_log_free(mapped_raw_log, filename))
```

Comment:
Should be `aeron_mapped_raw_log_free`.

Reply:
I'm doing some renaming and it should be `aeron_raw_log_free`.

Reply:
@mjpt777 Noticed that and already my branch.